integrationTests/vm/wasm/upgrades/upgrades_test.go

```diff
@@ -126,40 +126,6 @@ func TestUpgrades_ParentAndChildContracts(t *testing.T) {
 	require.Equal(t, uint64(42), context.QuerySCInt("getUltimateAnswer", [][]byte{}))
 }
 
-func TestUpgrades_UpgradeDelegationContract(t *testing.T) {
```

Comment:
why did we remove this test?

Reply:
It was, somehow, a duplicate (we already have a few tests that do test the upgrade flow). Plus, the delegation contract is the one written in Rust, plus it's a bit more difficult to find a version to be used in tests - then to add a version to the binary (filename) etc. So, removed for less testing bureaucracy, without actually harming the confidence in the upgrade flow.